setup.py

```diff
@@ -75,6 +75,8 @@
             "freezegun",
             "google-auth",
             "isort==5.10.1",
+            "jupyerlab",
+            "jupyterlab_server==2.19.0",
```

Comment:
why do you need server?

Reply:
This is something I am testing. Trying to fix dependency issues.